packages/react/src/_collapsing-side-bar/CollapsingSideBar.tsx

```diff
@@ -1,58 +1,58 @@
 import { PropsWithChildren, ReactNode, useRef } from 'react';
 import { useSpring, animated } from '@react-spring/web';
 import {
-	mapResponsiveProp,
-	mq,
+	packs,
 	tokens,
 	usePrefersReducedMotion,
 	useToggleState,
-	useWindowSize,
 } from '@ag.ds-next/react/core';
-import { backgroundColorMap, Box } from '@ag.ds-next/react/box';
+import { Box, type BoxProps } from '@ag.ds-next/react/box';
 import { BaseButton } from '@ag.ds-next/react/button';
 import { Flex } from '@ag.ds-next/react/flex';
-import { Stack } from '@ag.ds-next/react/stack';
 import { ChevronDownIcon } from '@ag.ds-next/react/icon';
+import { Stack } from '@ag.ds-next/react/stack';
+import { Text } from '@ag.ds-next/react/text';
 import {
-	hoverColorMap,
 	CollapsingSideBarBackground,
-	collapsingSideBarLocalPalette,
-	collapsingSideBarLocalPaletteVars,
+	collapsingSideBarHoverProp,
```

Comment:
Is this meant to be a `prop` imported from `utils`?

Reply:
Yeah. Open to better names. The original `LocalPalette` was weird too